backend.py

```diff
@@ -488,3 +497,10 @@ def set_robots_for_new_turn(state):
             robot.coordinates = robot.start_coordinates
             robot.damages = 0
             robot.direction = Direction.N
+
+def board_as_dict(map_name):
```

Comment:
Tato funkce načte mapu z json souboru, a převede tu načtenou strukturu zpátky na json. Je to asi zbytečný krok navíc - nebude lepší vzít rovnou tu mapu?

Reply:
a to napíšu prosím jak?

Reply:
`board_json = json.load(map_name)`?

Reply:
nebo jsou to přímo ty map_data v loadingu?